README.rst

```diff
@@ -0,0 +1,548 @@
+Dumb Password Rules
+===================
+
+Shaming sites with dumb password rules.
+
+Contributing
+------------
+
+Feel free to submit a pull request with dumb rules you've encountered.
+
+See other sites for the formatting and follow these rules:
+
+-  Include the name of the site with a link.
+-  Add a clean comment about the dumb password rule (optional).
+-  Include at least one screenshot.
+-  Keep the sites in alphabetical order.
+
+.. contents:: Table of Contents
```

Comment:
Can you make this stand out more? Maybe a header with `Table of Contents`. It currently looks like it's under the `Contributing` section.